org.kde.komodo.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "org.kde.komodo",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.9",
+  "sdk": "org.kde.Sdk",
+  "command": "komodo",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland"
+  ],
+  "cleanup": [
+    "*.a",
+    "*.la",
+    "/doc",
+    "/docs",
+    "/include",
+    "/lib/cmake",
+    "/lib/pkgconfig",
+    "/lib/*/cmake",
+    "/lib/*/pkgconfig",
+    "/man",
+    "/share/cmake",
+    "/share/doc",
+    "/share/man",
+    "/share/pkgconfig"
+  ],
+  "modules": [
+    {
+      "name": "kirigamiaddons",
+      "config-opts": [
+        "-DBUILD_TESTING=OFF"
+      ],
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.kde.org/stable/kirigami-addons/kirigami-addons-1.9.0.tar.xz",
```

Comment:
```suggestion
          "url": "https://cdn.download.kde.org/stable/kirigami-addons/kirigami-addons-1.9.0.tar.xz",
```

Reply:
Are you sure about this? We use download.kde.org links everywhere

Reply:
> Are you sure about this? We use download.kde.org links everywhere

```
❯ ds cdn.download.kde.org
1009236516.rsc.cdn77.org.
143.244.62.6
143.244.62.9

~ 
❯ ds download.kde.org
tinami.kde.org.
49.12.122.15
```

CDN goes to CDN77, without it goes to KDE infra.